recipes/python-cmethods/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "python_cmethods" %}
+{% set version = "2.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e03c78f610eedac605af1b60d58fdf81c1e9d7f1ee1b77b918e9256c71056ac2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - "cmethods = cmethods:cli"
+
+requirements:
+  build:
+    - python >=3.8
+    - setuptools_scm
```

Comment:
`setuptools`, `setuptools_scm` and `wheel` should all be under `host` requirements, and then you can get rid of the `requirements`/`build` section. It's typically only needed if you are compiling code. See:
https://conda-forge.org/docs/maintainer/adding_pkgs/#host
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host

Reply:
Thank you for sharing the relevant documentation!